server/src/main/java/org/opensearch/action/support/nodes/TransportNodesAction.java

```diff
@@ -209,6 +209,15 @@ protected void resolveRequest(NodesRequest request, ClusterState clusterState) {
         request.setConcreteNodes(Arrays.stream(nodesIds).map(clusterState.nodes()::get).toArray(DiscoveryNode[]::new));
     }
 
+    /**
+     * Return the concrete nodes from the request node ids which will be later used for routing requests to nodes.
+     **/
+    protected DiscoveryNode[] resolveConcreteNodes(NodesRequest request, ClusterState clusterState) {
```

Comment:
Can we call this getConcreteNodes? Since, we are just creating a list of nodes and returning from this method.